lambda/handlers/intentHandler.js

```
@@ -165,23 +584,31 @@ const NextPrayerTimeIntentHandler = {
             mosqueTimesData.jumua3,
           ];
           // Find the first non-null Jumu'ah time
-          const firstNonNullJumua = jumuaTimes.filter((time) => time !== null && time !== undefined).join(", ");
+          const firstNonNullJumua = jumuaTimes
+            .filter((time) => time !== null && time !== undefined)
+            .join(", ");
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)